frontends/web/src/routes/new-settings/appearance.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+import { View, ViewContent } from '../../components/view/view';
+import { Header, Main } from '../../components/layout';
+import { DarkmodeToggleSetting } from './components/settingsItems/darkmodeToggleSetting';
+import { DefaultCurrencySetting } from './components/settingsItems/defaultCurrencySetting';
+import { DisplaySatsToggleSetting } from './components/settingsItems/displaySatsToggleSetting';
+import style from './appearance.module.css';
+import { LanguageDropdownSetting } from './components/settingsItems/languageDropdownSetting';
+import { ActiveCurrenciesDropdownSetting } from './components/settingsItems/activeCurrenciesDropdownSetting';
+
+const ContentContainer = ({ children }: {children: ReactNode}) => <div className={style.contentContainer}>{children}</div>;
```

Comment:
I think `<ContentContainer>` is not really needed, and the style that does the margin belongs into settingsItemContaoiner.module.css